tracer/test/snapshots/AwsSqsTests.NetFramework.SchemaV0.verified.txt

```diff
@@ -8,7 +8,8 @@
     Tags: {
       env: integration_tests,
       language: dotnet,
-      runtime-id: Guid_1
+      runtime-id: Guid_1,
+      _dd.p.dm: -0
```

Comment:
It looks like `AwsSqsTests` doesn't have the XUnit attribute `[Trait("RunOnWindows", "True")]`, so I don't think this snapshot has been getting run! 🤯 

Looks like I should have one separate PR to fix these various snapshot testing oddities!

Reply:
I just now remembered why we don't run this on Windows: the SQS container is a Linux container! We're now running our Windows tests on our own Windows machines via Azure Scale Set, but it seems like it would be an undertaking, so I'll leave the AwsSqsTests alone for now